libraries/I2S/src/I2S.cpp

```diff
@@ -19,7 +19,7 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 #include <Arduino.h>
-#include "I2S.h"
+#include <I2S.h>
```

Comment:
Why the change from `"I2S.h"` to `<I2S.h>` ?  We want to only include the header in the same directory as this source file, so `""`s are appropriate here.